src/modules/user-preferences/application/use-cases/update-user-preferences.use-case.ts

```
@@ -0,0 +1,86 @@
+    }
+
+    if (updateDto.performance) {
+      preferences.performance = {
+        ...preferences.performance,
+        ...updateDto.performance,
+      };
+    }
+
+    try {
+      return await this.userPreferencesRepository.update(preferences);
+    } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)